tipg/resources/response.py

```diff
@@ -1,6 +1,29 @@
 """tipg custom responses."""
 
-from fastapi.responses import ORJSONResponse
+import decimal
+from typing import Any
+
+import orjson
+
+from fastapi.responses import JSONResponse
+
+
+def default(obj):
+    """Instruct orjson what to do with types it does not natively serialize"""
+    if isinstance(obj, decimal.Decimal):
+        return str(obj)
```

Comment:
why do we use `string` instead of `float` here? 

The result of this will return a string within the JSON so we loose the fact that it is a number in the table

cc @RemcoMeeuwissen @bitner

Reply:
I'm not actually sure what we want here as best practice as decimal/numeric can be arbitrarily larger than what float could handle

Reply:
Yeah string felt like the safe option to me so that's why I went with it, but I don't know at which point precision is lost so I don't know if it is actually a relevant concern.

Reply:
I'd say that we stick with string as it is the only way we can be certain to not be lossy. If someone knows their data fits into a float, they can alter their table.

Reply:
Looking at `simplejson` library, it seems they do convert decimal to float 
```python
a = {"a": decimal.Decimal('1.00000000006')}

simplejson.dumps(a)
'{"a": 1.00000000006}'
```

Reply:
🤔 I think it's because, simplejson will automatically convert decimal to float internally! 

> I'd say that we stick with string as it is the only way we can be certain to not be lossy. If someone knows their data fits into a float, they can alter their table.

👍 let's merge this, and we will revisit later if needed 🙏